build.sh

```diff
@@ -161,7 +155,7 @@ if [ "$xSTATIC_ANALYSIS" = "1" ] ; then
   CLANG_FORMAT=clang-format
   PEP8=pep8
   PERFORM_VERA=true
```

Comment:
I would add an empty newline here to separate the section that defines program names from one that defines flags. Also some comments on that the variables are would be helpful.